docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+  app:
+    image: node:10-alpine
```

Comment:
If the "end product" of this project is a Docker container, how about geting started on creating a Dockerfile for it - instead of defining configurations within Docker Compose?

Reply:
(copied from discord)

docker-compose will really shine when we add additional images, e.g. for postgres or redis or any other service we might need.  i think dockerfiles are really only necessary if we're adding special config to a base image (at the moment the base node10 image should be sufficient for what we need)

but also, a dockerfile is probably a good idea for setup including `npm install`

Reply:
> a dockerfile is probably a good idea for setup including npm install

Yes, in future we will add many dependencies which then require `npm install`. Can you please go ahead and create `dockerfile` and link to Docker Compose